### DISCUSSION TITLE
مشکل در اتصال به نو

### DISCUSSION THREAD

**Discussion by M3h64** [Gozargah/Marzban#619] [General]
درود. من یه مشکل دارم. وقتی تو پنل اصلی نودهارو با آیپی نود میزنم اتصال برقرار هست اما با دامین میزنم
Failed to connect  to nod's api 
میزنه 
توی سرو اوبنتو هم که مرزبان ریستارت میزنم ارور خاصی نشون نمیده

**Comment by abedin70** [comment]
> درود. من یه مشکل دارم. وقتی تو پنل اصلی نودهارو با آیپی نود میزنم اتصال برقرار هست اما با دامین میزنم Failed to connect to nod's api میزنه توی سرو اوبنتو هم که مرزبان ریستارت میزنم ارور خاصی نشون نمیده

تو مرزبان اصلا دامین نباید بزنی بنظرم

**Reply by M3h64** [reply]
خوب ینطوری که ایپی نود ها فیلتر بشه راه به راه باید با کاربرا بگی اپدیت کنن

**Reply by abedin70** [reply]
خوب یکی از دلایلی ک مرزبان نصب میکنی همینه دیگه
راحت با ی ابدیت همه چی ردیف میشه.
بهشون بگو هروت هیچکدوم از کانفیگا وصل نشد ی بروزرسانی اشتراک بزنن
